dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -325,3 +329,12 @@ def log_in_as(user)
   # Note: Safari webdriver does not support actions API
   @browser.action.key_down(:control).send_keys(key).key_up(:control).perform
 end
+
+And(/^I press keys "([^"]*)" for element "([^"]*)"$/) do |key, selector|
```

Comment:
Then I press keys "radius" for element "#domain-area input"

Sometimes we enter multiple keys